ibis/tests/sql/snapshots/test_sqlalchemy/test_no_cross_join/out.sql

```diff
@@ -1,15 +1,15 @@
 SELECT
-  t0.id AS id_x,
-  t0.personal,
-  t0.family,
-  t1.taken,
-  t1.person,
-  t1.quant,
-  t1.reading,
+  t3.id AS id_x,
+  t3.personal,
+  t3.family,
+  t4.taken,
+  t4.person,
+  t4.quant,
+  t4.reading,
   t2.id AS id_y,
   t2.site,
   t2.dated
-FROM person AS t0
+FROM person AS t3, survey AS t4, person AS t0
```

Comment:
Looks like this is still a problem. Back to the drawing board. Serious deja vu right now.

Reply:
The code for this is left unchanged.